oxy/yaml/wheel.yaml

```diff
@@ -0,0 +1,14 @@
+!Messages
+msgs:
+- !CANMsg
+    id: "0x680"
+    desc: "Wheel State"
+    fields: 
+    - !CANField
+        name: "WHEEL/Buttons/1"
```

Comment:
Uh doesn't the wheel message broadcast more than just 2 buttons? Isn't it bitwise encoded with like 10 this time around? Or are we waiting for wheel assembly/integration to fix this up?

Reply:
So what we do currently is have each button be one bit and then we decode it on the nero side. Now that we have ten buttons it heeds to he two messages for 10 bits